tensorflow_networking/gdr/Dockerfile

```diff
@@ -0,0 +1,84 @@
+ARG UBUNTU_VERSION=16.04
+
+FROM ubuntu:${UBUNTU_VERSION} AS base
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libhdf5-serial-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        pkg-config \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev \
+        openjdk-8-jdk \
+        openjdk-8-jre-headless \
+        libibverbs-dev \
+        librdmacm-dev \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV CI_BUILD_PYTHON python
+
+RUN git clone https://github.com/tensorflow/networking.git /tf_networking
```

Comment:
This would clone the master version of the code. Making this Dockerfile not usable for development/code-changes testing work, but more as a standalone tool.

See below suggestion for changes to make it usable for development